articles/install-guide/qjupyter.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Develop with Q# Jupyter notebooks
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.jupyter
+---
+
+# Develop with Q# Jupyter notebooks
+
+> [!WARNING]
```

Comment:
We don't want to have this warning as the first para. The first para should convey the intent of the article. It is what will be displayed as a preview in search results.

Reply:
Ok, I added a paragraph in 49a99a8 to display "Install" as first word and mantain uniformity with the other sections.